src/javascript/app/common/deriv_banner.js

```diff
@@ -0,0 +1,70 @@
+const getElementById = require('../../_common/common_functions').getElementById;
+const createElement  = require('../../_common/utility').createElement;
+
+const banner_types = {
+    rebranding: 'rebranding',
+    multiplier: 'multiplier',
+};
+
+const DerivBanner = (() => {
+    let el_rebranding_banner_countainer,
+        el_multiplier_banner_container,
+        el_banner_to_show,
+        el_close_button,
+        deriv_banner_type;
+
+    const onLoad = () => {
+        const is_deriv_banner_dismissed = localStorage.getItem('is_deriv_banner_dismissed');
+
+        if (!is_deriv_banner_dismissed) {
+            el_rebranding_banner_countainer = getElementById('deriv_banner_container');
+            el_multiplier_banner_container = getElementById('multiplier_banner_container');
+            deriv_banner_type = localStorage.getItem('deriv_banner_type');
+
+            showBanner();
+            el_close_button = el_banner_to_show.querySelector('.deriv_banner_close') || createElement('div');
+            el_close_button.addEventListener('click', onClose);
+        }
+    };
+
+    const onClose = () => {
+        el_banner_to_show.setVisibility(0);
+        localStorage.setItem('is_deriv_banner_dismissed', 1);
+    };
+
+    const showBanner = () => {
+        if (deriv_banner_type === banner_types.rebranding) {
```

Comment:
what if the localstorage item is not set yet? then we should set it first using the 50% random function, right?